### ISSUE TITLE
%N is not supported for date format

### ISSUE THREAD

**Issue by Flarescape** [sebastienros/fluid#181]
It would be great if it's possible to add support for %N (fractional seconds digits), please.
This would be great to generate random numbers!

**Comment by sebastienros** [comment]
Can you point me to some example? Couldn't find it in ruby date formats.

**Comment by Flarescape** [comment]
Fractional seconds digits - https://ruby-doc.org/stdlib-2.6.1/libdoc/date/rdoc/DateTime.html
I hope, this is something default and not some extension.
Else I'm sorry!

**Comment by sebastienros** [comment]
Is that really necessary since `%L` already gives the milliseconds, and `%N` will only be able to append zeros after that?

**Comment by Flarescape** [comment]
Actually not really, no!
This was just my idea for a "liquid only" random number.
I encountered some situations, where this would be helpfull, but of course I could just build my own liquid helper for it or use a cshtml template instead.
I'm sorry for bothering you with this!
Since I've created this issue, I've gained much more knowledge of OC and everything around it and should have closed this by now!
Thank you anyway for asking! 👍
